src/test/java/com/vaadin/testbenchexample/junit5/AbstractJUnit5IT.java

```diff
@@ -20,7 +22,11 @@ public abstract class AbstractJUnit5IT extends BrowserTestBase implements Driver
 
     @Override
     public WebDriver createDriver() {
-        return new ChromeDriver();
+        ChromeOptions options = new ChromeOptions();
+        if (Boolean.getBoolean("com.vaadin.testbench.Parameters.headless")) {
+            options.addArguments("--headless");
```

Comment:
Flow uses `--headless=new` to set up headless tests

https://developer.chrome.com/articles/new-headless/

Reply:
done, thanks